packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/brainTreeDropIn.js

```diff
@@ -52,6 +52,7 @@ const BraintreeDropin = props => {
                 overrides: {
                     fields: {
                         number: {
+                            placeholder: '16-Digit Number',
```

Comment:
For internationalisation all strings should be translatable.

Reply:
done, @chris-brabender have a look pls